src/behaviours/helm-releases/showing-details-for-helm-release.test.ts

```diff
@@ -65,25 +81,30 @@ describe("showing details for helm release", () => {
       );
 
       rendererDi.override(
-        callForHelmReleasesInjectable,
-        () => callForHelmReleasesMock,
+        requestHelmReleasesInjectable,
+        () => requestHelmReleasesMock,
       );
 
       rendererDi.override(
-        callForHelmReleaseInjectable,
-        () => callForHelmReleaseMock,
+        requestHelmReleaseInjectable,
+        () => requestHelmReleaseMock,
       );
 
       rendererDi.override(
-        callForHelmReleaseConfigurationInjectable,
-        () => callForHelmReleaseConfigurationMock,
+        requestHelmReleaseConfigurationInjectable,
+        () => requestHelmReleaseConfigurationMock,
       );
 
       rendererDi.override(
-        callForHelmReleaseUpdateInjectable,
-        () => callForHelmReleaseUpdateMock,
+        requestHelmReleaseUpdateInjectable,
+        () => requestHelmReleaseUpdateMock,
       );
 
+      rendererDi.override(requestHelmChartsInjectable, () => requestHelmChartsMock);
```

Comment:
These overrides are not used for anything. They should probably be global overrides instead?

Reply:
Sure, but in this specific case I used them in a future commit to fix the tests.